src/pages/cohorts/bpd.astro

```diff
@@ -131,43 +88,33 @@ import Syllabus from "../../components/cohortPages/Syllabus.astro";
   syllabus={[
     {
       week: "Week 01",
-      name: "Introductions",
+      name: "Welcome to the Bitcoin Protocol Development Seminar",
       chNo: "",
```

Comment:
Can we add chapter 1, 2, 3 etc or else we will have a blank column?
<img width="434" alt="Screenshot 2023-12-19 at 11 35 52 PM" src="https://github.com/Bitshala/website-v2/assets/32434322/5f70883a-6dd6-4705-b01b-e88f2fa53d0b">